libslide/src/evaluator_rules/pattern_matching.rs

```diff
@@ -16,8 +24,8 @@ pub fn match_rule(rule: Expr, target: Expr) -> Option<Replacements> {
             }
```

Comment:
This can be simplified to 
if a = b {
Some(Replacements::default())
}
none

Reply:
fixed